modules/flash-script.nix

```diff
@@ -50,6 +49,55 @@ in
             type = types.listOf types.path;
             default = [];
           };
+
+          capsuleAuthentication = {
+            enable = mkEnableOption "capsule update authentication";
+
+            publicCertificateDer = mkOption {
```

Comment:
Recommend renaming these options to end with "File", to match what is used for the secure boot keys.